app/models/article_result.rb

```diff
@@ -3,10 +3,16 @@
 class ArticleResult
   FORMAT_TO_ICON = {
     'Academic Journal' => 'book-open-4.svg',
+    'Article' => 'wrap-text-around.svg',
+    'Audio' => 'microphone-3.svg',
     'Books' => 'notebook.svg',
+    'eBooks' => 'notebook.svg',
+    'Maps' => 'map-pin-1.svg',
+    'Music scores' => 'file-music-1.svg',
     'News' => 'newspaper.svg',
-    'eBook' => 'notebook.svg',
-    'Video Recording' => 'camera-film-1.svg'
+    'Report' => 'notepad-1.svg',
+    'Video Recording' => 'camera-film-1.svg',
+    'Videos' => 'camera-film-1.svg'
```

Comment:
How are you finding these?  When I filter by the facet that says "Videos" and then I look at the returned documents I see "Video Recording"

Reply:
We're using the values from the Source Type facet (`eds_publication_type`) to drive this list. I actually haven't seen a search that has `Video Recording`, but the data is provider-specific so maybe Niqui found it somewhere.